DXRModules/DeusEx/Classes/DXRMissions.uc

```diff
@@ -1193,7 +1232,19 @@ function bool MoveActor(Actor a, vector loc, rotator rotation, EPhysics p)
         sp.HomeLoc = sp.Location;
         sp.HomeRot = vector(sp.Rotation);
         sp.DesiredRotation = rotation;
+        if (!sp.bInWorld){
+            sp.WorldPosition = sp.Location;
+            sp.SetLocation(sp.Location+vect(0,0,20000));
+        }
     }
+    v = Vehicles(a);
+    if ( v != None ) {
+        if (!v.bInWorld){
+            v.WorldPosition = v.Location;
+            v.SetLocation(v.Location+vect(0,0,20000));
```

Comment:
I feel like we just need to assign the new goal location to the WorldLocation variable, and nothing else matters?